DigitalLearningSolutions.Web/Views/Register/Password.cshtml

```diff
@@ -0,0 +1,61 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.Register
+@model PasswordViewModel
+
+@{
+  var errorHasOccurred = !ViewData.ModelState.IsValid;
+  ViewData["Title"] = errorHasOccurred ? "Error: Register - Password": "Register - Password";
+  var passwordError = ViewData.ModelState["Password"]?.Errors?.Count > 0;
+  var confirmPasswordError = ViewData.ModelState["ConfirmPassword"]?.Errors?.Count > 0;
+  var passwordInputErrorClass = passwordError ? "nhsuk-select--error" : "";
+  var passwordFormErrorClass = passwordError ? "nhsuk-form-group--error" : "";
+  var confirmPasswordInputErrorClass = confirmPasswordError ? "nhsuk-select--error" : "";
+  var confirmPasswordFormErrorClass = confirmPasswordError ? "nhsuk-form-group--error" : "";
+}
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-full">
+    <form method="post" asp-action="Password">
+
+      @if (errorHasOccurred) {
+        <partial name="_ErrorSummary"/>
+      }
+
+      <div class="nhsuk-u-margin-bottom-8">
+        <h1 id="page-heading">Create password</h1>
+      </div>
+
+      <div class="nhsuk-form-group @passwordFormErrorClass">
+        <label class="nhsuk-label" for="Password">
+          Password
+        </label>
+        <div class="nhsuk-hint" id="password-hint">
+          Your new password should have a minimum of 8 characters with at least 1 letter, 1 number and 1 symbol.
+        </div>
+        @if (passwordError) {
+          <span class="nhsuk-error-message" id="password-error">
+            <span class="nhsuk-u-visually-hidden">Error:</span> @ViewData.ModelState["Password"].Errors[0].ErrorMessage
+          </span>
+        }
+        <input class="nhsuk-input nhsuk-u-width-one-half @passwordInputErrorClass" id="Password" spellcheck="false" autocomplete="new-password" aria-describedby="password-error password-hint" asp-for="Password"/>
+      </div>
+
+      <div class="nhsuk-form-group @confirmPasswordFormErrorClass">
+        <label class="nhsuk-label" for="ConfirmPassword">
+          Re-type password
+        </label>
+        @if (confirmPasswordError) {
+          <span class="nhsuk-error-message" id="confirm-password-error">
+            <span class="nhsuk-u-visually-hidden">Error:</span> @ViewData.ModelState["ConfirmPassword"].Errors[0].ErrorMessage
+          </span>
+        }
+        <input class="nhsuk-input nhsuk-u-width-one-half @confirmPasswordInputErrorClass" id="ConfirmPassword" spellcheck="false" autocomplete="new-password" aria-describedby="confirm-password-error" asp-for="ConfirmPassword"/>
+      </div>
+
+      <button class="nhsuk-button" type="submit">Next</button>
+    </form>
+
+    <p>
+      <vc:back-link asp-controller="Register" asp-action="learner-information" />

```

Comment:
Not sure about marking the back link as paragraph in HTML syntax - is this a weird accessibility thing? 😮